internal/common/types.go

```diff
@@ -250,6 +261,20 @@ func getDistributionMapping() map[string]int {
 	return mapping
 }
 
+func (dist *Distribution) ModulePlatformID() (string, error) {
+	mapping := map[Distribution]string{
+		// TODO: this could be refactored so we don't have these strings hard coded in two places
+		Fedora30: "platform:f30",
+		RHEL82:   "platform:el8",
+	}
+	id, exists := mapping[*dist]
+	if !exists {
+		return "", &CustomTypeError{reason: "Distribution does not have a module platform ID assigned"}
```

Comment:
I think we should just panic here, this is clearly our mistake.

Reply:
Makes sense!

Reply:
Thinking about it again, I think we should keep it like this because we have the same logic in other transformation functions in this package.

Reply:
I would have preferred to look up the distro and query that, but let's not block on that. Not a fan of hardcoding strings more than once.